sdk/v1/client.go

```diff
@@ -100,19 +100,21 @@ func (cli *Client) PostTasksCount(ctx context.Context, headers client.Headers, j
 
 	path := fmt.Sprintf("%s/jobs/%s/tasks", cli.apiVersion, jobID)
 
-	_, b, err := cli.callReindexAPI(ctx, path, http.MethodPost, headers, payload)
+	respHeader, b, err := cli.callReindexAPI(ctx, path, http.MethodPost, headers, payload)
 	if err != nil {
-		return task, err
+		return "", task, err
 	}
 
 	if err = json.Unmarshal(b, &task); err != nil {
-		return task, apiError.StatusError{
+		return "", task, apiError.StatusError{
 			Err:  fmt.Errorf("failed to unmarshal bytes into reindex job, error is: %v", err),
 			Code: http.StatusInternalServerError,
 		}
 	}
 
-	return task, nil
+	respETag := respHeader.Get(ETagHeader)
+
+	return respETag, task, nil
```

Comment:
(suggestion): Instead of returning 3 parameters , can the tag be part of a metadata object of the task model ?

Reply:
part of the existing implementation, leaving atm for wider discussion.